src/meta/service/src/ddl_service.rs

```diff
@@ -882,6 +882,7 @@ impl DdlService for DdlServiceImpl {
                     .into_iter()
                     .map(|id| id as _)
                     .collect(),
+                true,
```

Comment:
Shall we also include the parameter `include_dropped_table` in `GetTableRequest` to make this rpc call more general?